website/content/api-docs/acl/auth-methods.mdx

```diff
@@ -0,0 +1,392 @@
+    configured with your OIDC provider.
+
+  - `BoundAudiences` `(array<string>)` - List of aud claims that are valid for
+    login; any match is sufficient.
+
+  - `AllowedRedirectURIs` `(array<string>)` - A list of allowed values for
+    redirect_uri. Must be non-empty.
+  
+  - `DiscoveryCaPem` `(array<string>)` - PEM encoded CA certs for use by the TLS
+    client used to talk with the OIDC Discovery URL. If not set, system
+    certificates are used.
+
+  - `SigningAlgs` `(array<string>)` - A list of supported signing algorithms.
+    Defaults to `RS256`.
+
+  - `ClaimMappings` `(map[string]string)` - Mappings of claims (key) that will
+    be copied to a metadata field (value). Use this if the claim you are capturing
+    is singular (such as an attribute).
+
+    When mapped, the values in each list can be any of a number, string, or
+    boolean and will all be stringified when returned.
+
+  - `ListClaimMappings` `(map[string]string)` - Mappings of claims (key) will be
+    copied to a metadata field (value). Use this if the claim you are capturing is
+    list-like (such as groups).
+
+### Sample Payload
+
+```json
+{
+    "name": "example-acl-auth-method",
+    "type": "OIDC",
+    "tokenlocality": "global",
+    "maxtokenttl": "1h0m0s",
+    "default": true
+}
+```
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --request POST \
+    --header "X-Nomad-Token: <NOMAD_TOKEN_SECRET_ID>" \
+    --data @payload.json \
+    https://localhost:4646/v1/acl/auth-method/example-acl-auth-method
+```
+
+### Sample Response
+
+```json
+{
+    "MaxTokenTTL": "1h0m0s",
+    "Name": "example-acl-auth-method",
+    "Type": "OIDC",
+    "TokenLocality": "global",
+    "Default": true,
+    "Config": null,
+    "CreateTime": "0001-01-01T00:00:00Z",
+    "ModifyTime": "0001-01-01T00:00:00Z",
+    "CreateIndex": 12,
+    "ModifyIndex": 32
+}
+```
+
+## List Auth Methods
+
+This endpoint lists all ACL Auth Methods. This lists the auth methods that have
+been replicated to the region, and may lag behind the authoritative region.
+
+| Method | Path                | Produces           |
+| ------ | ------------------- | ------------------ |
+| `GET`  | `/acl/auth-methods` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs#blocking-queries),
+[consistency modes](/api-docs#consistency-modes) and
+[required ACLs](/api-docs#acls).
+
+| Blocking Queries | Consistency Modes | ACL Required                                                                                                                             |
+| ---------------- | ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| `YES`            | `all`             | `management` for all auth methods.<br />Output when given a non-management token will be limited to the auth methods on the token itself |
```

Comment:
I believe this is incorrect as we don't know which auth method created the token and provide this functionality.

Reply:
Right! we don't require an ACL for `List`, we just return stubs anyway.